### ISSUE TITLE
Feature request: use mouse drag to reset the in/out cutpoint

### ISSUE THREAD

**Issue by zhuoziqiang** [mifi/lossless-cut#517]
sometimes dragging the mouse is better than shortcut keys. just like many other video editing software

**Comment by mifi** [comment]
I agree but it could be annoying if the user tries to drag the timeline marker but instead the accidentally drag the cutpoint. maybe need to drag using a modifier key.

**Comment by markusd1984** [comment]
Or perhaps require 2-3 seconds hovering over a segment marker and only then show double pointers to be able to move segment in/out points, that way one can still move timeline marker without risk moving segments.

**Comment by markusd1984** [comment]
**handle bars** could be displayed for a few seconds above the timeline during hover over segments (or permanently).

**Comment by Gunstick** [comment]
Is it currently possible in some way to move cut points? I just added a feature request for that before finding this one.

**Comment by mifi** [comment]
@Gunstick  yes with I/O or the buttons. See help

**Comment by mifi** [comment]
mouse drag resize/move segments implemented. can be tested in next nightly
